frontend/src/metabase/containers/Form.jsx

```diff
@@ -232,7 +232,11 @@ export default class Form extends React.Component {
         };
       } else if (error) {
         throw {
-          _error: error.data.message || error.data,
+          _error:
```

Comment:
Error order updated:
1. `message` from `error.data`
2. `Error.message`
3. `error.data` (might be a string?)
4. default error message